docs(deployment): add deployment chapter (#494)
instill-ai/instill.tech: docs/deployment/overview.mdx

---
title: "Deployment Overview"
lang: "en-US"
draft: false
description: "Learn about how to deploy open-source VDP https://github.com/instill-ai/vdp"
---

> Versatile Data Pipeline (VDP) provides AI and data practitioners a infrastructure to build up complex pipelines to

Reviewer: ```suggestion
Versatile Data Pipeline (VDP) provides AI and data practitioners an infrastructure to build up complex pipelines to
```

> concatenate data connectors and AI models to process unstructured data.

Reviewer: ```suggestion
extract unstructured data from data connectors and process it with AI models.
```
